public/index.html

```diff
@@ -3,6 +3,8 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Welcome to My Portfolio - showcasing my skills and projects.">
+    <meta name="keywords" content="portfolio, web development, skills, contact information">
```

Comment:
It would be nice to add your name in both description and keyword. Also, please try to make the description little more descriptive. Maybe you can use chatGPT's help.